docs/source/conf.py

```diff
@@ -3,6 +3,14 @@
 
 import sphinx_rtd_theme
 
+import importlib.util
+spec = importlib.util.spec_from_file_location("generate_example_images", "../generate_example_images.py")
```

Comment:
Add some comments here to indicate that the doc build happens in an isolated container, so that we need to include the preprocessing part manually here.